specification_templates/forecast.yaml

```diff
@@ -1,15 +1,16 @@
 data:
-  regression_version: '2021_02_10.14'
-  covariate_version: '2021_02_10.04'
+  regression_version: '2021_02_15.05'
+  covariate_version: '2021_02_11.01'
   output_root: ''
+  output_format: 'parquet'
   fh_subnationals: False
 workflow:
   project: 'proj_covid_prod'
   queue: 'all.q'
   tasks:
     beta_residual_scaling:
       max_runtime_seconds: 5000
-      m_mem_free: '5G'
+      m_mem_free: '50G'
```

Comment:
I'm all for throwing memory at the problem but I have to ask - did this actually cause a 10x memory usage increase or did you just pick a big number so it wouldn't oom?

If so, what's the real number?

Reply:
On my to do list.  Parquet does compression and so need more mem to load from disk.  Just 10xed it for now since it's unlikely for me to find 26 free threads without 50G of mem on a node.